src/SprykerEco/Yves/Adyen/Theme/default/components/molecules/adyen-credit-card/adyen-credit-card.twig

```diff
@@ -0,0 +1,42 @@
+{% extends model('component') %}
+
+{% define config = {
+    name: 'adyen-credit-card',
+    tag: 'adyen-credit-card'
```

Comment:
Add comma at the end. Check it everywhere.

Reply:
fixed